az-flow-trigger-dependency-plugin/build.gradle

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Please add a readme as an instruction for the implementation development.

Reply:
updated